reconcile/external_resources/integration.py

```diff
@@ -65,23 +96,28 @@ def run(
         er_inventory=er_inventory,
         module_inventory=m_inventory,
         state_manager=ExternalResourcesStateDynamoDB(
+            aws_api=get_aws_api(
+                query_func=gql.get_api().query,
+                account_name=er_settings.state_dynamodb_account.name,
+                region=er_settings.state_dynamodb_region,
+                secret_reader=secret_reader,
+            ),
```

Comment:
`aws_api` need to close session after use, can either use `with` or `defer` to call `aws_api.close`

Reply:
Added a context manager. PTAL